src/views/Settings.vue

```diff
@@ -280,6 +280,32 @@
               </b-modal>
             </div>
           </div>
+          <div class="pt-0">
+            <div class="d-flex w-100 justify-content-between px-3 px-lg-4 mb-4">
+              <div>
+                <span class="d-block">Debug</span>
+                <small class="d-block" style="opacity: 0.4">Run the debug script</small>
+              </div>
+              <b-button variant="outline-warning" size="sm" @click="debugPrompt">Debug</b-button>
+              <b-modal
+                ref="debug-modal"
+                title="Results"
+                no-close-on-backdrop
+                no-close-on-esc
+                cancel-title="Run again"
+                @cancel="debugPrompt"
+                @ok="clearDebugInterval"
+              >
+                <div v-if="this.loadingDebug">
+                  <p>Processing...</p>
+                </div>
+                <div v-else>
+                    <p>Please share the following links with a description of your problem in the <a href="https://t.me/getumbrel">Umbrel Telegram group</a> so we can help you.</p>
+                    <input-copy class="mb-1" size="sm" auto-width :value="this.debugResult.linkDebug"></input-copy>
```

Comment:
Can you center this? The following change could do it.

```suggestion
                    <input-copy class="mb-1 mx-auto" size="sm" auto-width :value="this.debugResult.linkDebug"></input-copy>
```